license_manager/apps/subscriptions/signals/receivers.py

```diff
@@ -39,13 +40,15 @@ def send_event_to_message_bus(**kwargs):  # pragma: no cover
         license_event_data = {"license": kwargs['license']}
         message_key = license_event_data['license'].enterprise_customer_uuid
         event_type = kwargs['signal'].event_type
-
+        logger.info(f"Producing event for license with uuid {license_event_data['license'].license_uuid} to event bus")
         license_event_producer.produce(settings.LICENSE_TOPIC_NAME, key=message_key,
                                        value=license_event_data, on_delivery=verify_event,
                                        headers={EVENT_TYPE_HEADER_KEY: event_type})
         license_event_producer.poll()
     except ValueSerializationError as vse:
         logger.exception(vse)
+    except Exception as e:  # pylint: disable=broad-except
+        logger.exception(e)
```

Comment:
Update to something like the following.
* Logging exception will automatically included the exception details.
* Provide some standard message for you to search for in logs.
```suggestion
    except Exception:  # pylint: disable=broad-except
        logger.exception('Unknown error producing license event.")
```